java-connectors/kafka-connect-azure-eventhubs/src/main/java/io/lenses/java/streamreactor/connect/azure/eventhubs/config/AzureEventHubsConfig.java

```diff
@@ -1,154 +1,185 @@
+            //).define(AzureEventHubsConfigConstants.SCHEMA_GROUP,
+            //    Type.STRING,
+            //    AzureEventHubsConfigConstants.OPTIONAL_EMPTY_DEFAULT,
+            //    Importance.MEDIUM,
+            //    AzureEventHubsConfigConstants.SCHEMA_GROUP_DOC,
+            //    CONNECTION_GROUP,
+            //    6,
+            //    ConfigDef.Width.MEDIUM,
+            //    AzureEventHubsConfigConstants.SCHEMA_GROUP
+            //).define(AzureEventHubsConfigConstants.CLIENT_ID,
+            //    Type.STRING,
+            //    AzureEventHubsConfigConstants.OPTIONAL_EMPTY_DEFAULT,
+            //    Importance.HIGH,
+            //    AzureEventHubsConfigConstants.CLIENT_ID_DOC,
+            //    CONNECTION_GROUP,
+            //    7,
+            //    ConfigDef.Width.MEDIUM,
+            //    AzureEventHubsConfigConstants.CLIENT_ID
+            //).define(AzureEventHubsConfigConstants.CLIENT_SECRET,
+            //    Type.STRING,
+            //    AzureEventHubsConfigConstants.OPTIONAL_EMPTY_DEFAULT,
+            //    Importance.HIGH,
+            //    AzureEventHubsConfigConstants.CLIENT_SECRET_DOC,
+            //    CONNECTION_GROUP,
+            //    8,
+            //    ConfigDef.Width.MEDIUM,
+            //    AzureEventHubsConfigConstants.CLIENT_SECRET
+            //).define(AzureEventHubsConfigConstants.TENANT_ID,
+            //    Type.STRING,
+            //    AzureEventHubsConfigConstants.OPTIONAL_EMPTY_DEFAULT,
+            //    Importance.LOW,
+            //    AzureEventHubsConfigConstants.TENANT_ID_DOC,
+            //    CONNECTION_GROUP,
+            //    9,
+            //    ConfigDef.Width.MEDIUM,
+            //    AzureEventHubsConfigConstants.TENANT_ID
+            //).define(AzureEventHubsConfigConstants.USER_INFO,
+            //    Type.STRING,
+            //    AzureEventHubsConfigConstants.OPTIONAL_EMPTY_DEFAULT,
+            //    Importance.LOW,
+            //    AzureEventHubsConfigConstants.USER_INFO_DOC,
+            //    CONNECTION_GROUP,
+            //    10,
+            //    ConfigDef.Width.MEDIUM,
+            //    AzureEventHubsConfigConstants.USER_INFO
+        ).define(AzureEventHubsConfigConstants.KCQL_CONFIG,
+            Type.STRING,
+            AzureEventHubsConfigConstants.KCQL_DEFAULT,
+            Importance.HIGH,
+            AzureEventHubsConfigConstants.KCQL_DOC,
+            "Mappings",
+            1,
+            ConfigDef.Width.LONG,
+            AzureEventHubsConfigConstants.KCQL_CONFIG
+        ).define(AzureEventHubsConfigConstants.INCLUDE_HEADERS,
+            Type.BOOLEAN,
+            AzureEventHubsConfigConstants.INCLUDE_HEADERS_DEFAULT,
+            Importance.MEDIUM,
+            AzureEventHubsConfigConstants.INCLUDE_HEADERS_DOC,
+            "Mappings",
+            2,
+            ConfigDef.Width.LONG,
+            AzureEventHubsConfigConstants.INCLUDE_HEADERS
+        ).define(
```

Comment:
The nbr retries is from the old approach for handling failure and have the retry in the connector. With Connect framework improved retries , this option is not required.

Reply:
Removed, thanks!